FinniversKit/Sources/Components/FrontPageTransactionView/FrontPageTransactionViewModel.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct FrontPageTransactionViewModel {
+    public let headerTitle: String
+    public let title: String
+    public let subtitle: String
+    public let imageUrl: String?
+    public let adId: Int?
+    public let transactionId: String?
+
+    public init(headerTitle: String,
+                title: String,
+                subtitle: String,
+                imageUrl: String? = nil,
+                adId: Int? = nil,
+                transactionId: String? = nil
+                ) {
```

Comment:
Formatting for consistency😄

```suggestion
    public init(
        headerTitle: String,
        title: String,
        subtitle: String,
        imageUrl: String? = nil,
        adId: Int? = nil,
        transactionId: String? = nil
    ) {
```

Reply:
done ☺️